apps/src/bin/namada-relayer/main.rs

```diff
@@ -14,5 +14,6 @@ async fn main() -> Result<()> {
 
     let cmd = cli::namada_relayer_cli()?;
     // run the CLI
-    CliApi::<CliIo>::handle_relayer_command::<HttpClient>(None, cmd).await
+    CliApi::<CliIo>::handle_relayer_command::<HttpClient>(None, cmd, &CliIo)
```

Comment:
It may be way more work than is worth it right now, but does this mean we can get rid of the explicit generic param?

Reply:
Yes, that should be the case. Let me try removing the explicit generic parameter now...